tools/bismark/input/bisulfite-seq_test_data.md

```diff
@@ -0,0 +1 @@
+../../../general/fastq/methylated_dna/bisulfite-seq_test_data.md
```

Comment:
I don't think you need to softlink the readme in here, just the fastqs is fine..

Reply:
Didn't you specifically ask for that further upwards?

Reply:
> Readme file also in general/fastq/methylated_dna/ (alongside the files)

Reply:
Alongside the files, yeah - this is alongside the softlinks. I was kind of thinking that people should be able to follow the softlinks and figure it out, but I guess it doesn't really matter at all.